chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -322,6 +322,9 @@ def find_node_boot_mac_addresses(node, admin_data_net)
             cloud_available = true if name.include? "Cloud"
           end
 
+          ntp_servers = search(:node, "roles:ntp-server")
```

Comment:
`search_env_filtered` (and `get_instance`) should only be used when a proposal references another proposal through its attributes. It's not the case here, as the provisioner proposal doesn't reference the ntp proposal.